src/Databases/DatabaseReplicatedWorker.cpp

```diff
@@ -32,6 +32,9 @@ DatabaseReplicatedDDLWorker::DatabaseReplicatedDDLWorker(DatabaseReplicated * db
 
 bool DatabaseReplicatedDDLWorker::initializeMainThread()
 {
+    initialization_duration_timer.restart();
+    initializing.store(true, std::memory_order_release);
```

Comment:
This is not a performance-critical part of code, so it's okay to use the default `std::memory_order_seq_cst`. Using anything besides `std::memory_order_seq_cst` and `std::memory_order_relaxed` usually requires a comment explaining why it will work correctly, and I would prefer not to think about this when it's not required for performance reasons

Reply:
See also https://github.com/ClickHouse/ClickHouse/pull/67312/files :)